### DISCUSSION TITLE
Command center feature/client

### DISCUSSION THREAD

**Discussion by DaredevilFD** [SafeExamBrowser/seb-win-refactoring#255] [Ideas]
Entirely possible you've had this suggested before, but I'm throwing in my two cents.

In order for SEB to be a fully fledged testing system i feel the command center is missing.
A separate client within the same network as the users, that allows you to arrange rooms and administrate those users within the rooms. Ideally with features that would allow you to get an overview of progress and activity-logging. Pausing timers at ongoing tests for entire rooms/groups or individual users. Any and all information within reason, really.

Whether or not this is possible with SEB, I dont know.
Regardless, you guys rock!

**Comment by dbuechel** [comment]
Thanks a lot for your input. Yes indeed, there exists the [SEB-Server](https://github.com/SafeExamBrowser/seb-server#readme) which can be used to centrally manage and control SEB clients. It already has quite an extensive [user guide](https://seb-server.readthedocs.io/en/latest/overview.html) which will provide you with a good overview over its current features and capabilities.
